metron-interface/metron-alerts/src/app/alerts/configure-table/configure-table.component.ts

```diff
@@ -165,21 +168,18 @@ export class ConfigureTableComponent implements OnInit, AfterViewInit {
   }
 
   save() {
-    let selectedColumns = this.allColumns.filter((mDataWrapper: ColumnMetadataWrapper) => mDataWrapper.selected)
-                          .map((mDataWrapper: ColumnMetadataWrapper) => mDataWrapper.columnMetadata);
-
-    this.configureTableService.saveColumnMetaData(selectedColumns).subscribe(() => {
-      this.saveColumnNames();
-    }, error => {
-      console.log('Unable to save column preferences ...');
-      this.saveColumnNames();
-    });
-
-
+    this.configureTableService.saveColumnMetaData(
+      this.visibleColumns.map(columnMetaWrapper => columnMetaWrapper.columnMetadata))
+      .subscribe(() => {
+        this.saveColumnNames();
+      }, error => {
+        console.log('Unable to save column preferences ...');
```

Comment:
shouldn't it be removed?

Reply:
I haven't touched this part of the code, only a few lines above. By theory, we could remove console.log statements or bind them to isDevMode(), but hence console.logs are everywhere in the code I think we just make it inconsistent.